src/lib/spinel/spinel.h

```diff
@@ -2944,6 +2984,160 @@ enum
+     *   | Link margin   |  2 | `C` (uint8_t)  |
+     *   | RSSI          |  3 | `c` (int8_t)   |
+     *   +---------------+----+----------------+
+     *
+     */
+    SPINEL_PROP_THREAD_LINK_METRICS_QUERY_RESULT = SPINEL_PROP_THREAD_EXT__BEGIN + 46,
+
+    /// Link metrics probe
+    /** Format `6CC` - Write only
+     * Required capability: `SPINEL_CAP_THREAD_LINK_METRICS`
+     *
+     * Send a MLE Link Probe message to the peer.
+     *
+     * `6` : IPv6 destination address
+     * `C` : The Series ID for which this Probe message targets at
+     * `C` : The length of the Probe message, valid range: [0, 64]
+     *
+     */
+    SPINEL_PROP_THREAD_LINK_METRICS_PROBE = SPINEL_PROP_THREAD_EXT__BEGIN + 47,
+
+    /// Link metrics Enhanced-ACK Based Probing management
+    /** Format: 6Cd - Write only
+     *
+     * Required capability: `SPINEL_CAP_THREAD_LINK_METRICS`
+     *
+     * `6` : IPv6 destination address
+     * `C` : Indicate whether to register or clear the probing. `0` - clear, `1` - register
+     * `d` : List of requested metric ids encoded as single bytes (`C`)
+     *
+     *   +---------------+----+
+     *   |    Metric     | Id |
+     *   +---------------+----+
+     *   | LQI           |  1 |
+     *   | Link margin   |  2 |
+     *   | RSSI          |  3 |
+     *   +---------------+----+
+     *
+     * Result of configuration is reported asynchronously to the Host using the
+     * @ref SPINEL_PROP_THREAD_LINK_METRICS_MGMT_RESPONSE.
+     *
+     * Whenever Enh-ACK IE report is received it is passed to the Host using the
+     * @ref SPINEL_PROP_THREAD_LINK_METRICS_MGMT_ENH_ACK_IE property.
+     *
+     */
+    SPINEL_PROP_THREAD_LINK_METRICS_MGMT_ENH_ACK = SPINEL_PROP_THREAD_EXT__BEGIN + 48,
+
+    /// Link metrics Enhanced-ACK Based Probing IE report
+    /** Format: SEA(t(CD)) - Unsolicited notifications only
+     *
+     * Required capability: `SPINEL_CAP_THREAD_LINK_METRICS`
+     *
+     * `S` : Short address of the Probing Subject
+     * `E` : Extended address of the Probing Subject
+     * `A(t(CD))` : Array of structs encoded as following:
+     *   `C` : Metric id
+     *   `D` : Metric value
+     *
+     *   +---------------+----+----------------+
+     *   |    Metric     | Id |  Value format  |
+     *   +---------------+----+----------------+
+     *   | LQI           |  1 | `C` (uint8_t)  |
+     *   | Link margin   |  2 | `C` (uint8_t)  |
+     *   | RSSI          |  3 | `c` (int8_t)   |
+     *   +---------------+----+----------------+
+     *
+     */
+    SPINEL_PROP_THREAD_LINK_METRICS_MGMT_ENH_ACK_IE = SPINEL_PROP_THREAD_EXT__BEGIN + 49,
+
+    /// Link metrics Forward Tracking Series management
+    /** Format: 6Cdd - Write only
+     *
+     * Required capability: `SPINEL_CAP_THREAD_LINK_METRICS`
+     *
+     * `6` : IPv6 destination address
+     * `C` : Series id
+     * `d` : List of tracked frame types encoded as single bytes (`C`), if the list is empty
```

Comment:
When transmitting this field in Link Metrics Management Request, we use bit fields in one byte to represent this information: [SeriesFlags](https://github.com/openthread/openthread/blob/master/src/core/thread/link_metrics_tlvs.hpp#L360)  

Shall we use one byte to replace a list (data blob) here?